client/app/react-components/DestinationOption.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Destination from './Destination';
+
+export default class DestinationOption extends React.Component {
+  handleMouseDown = (event) => {
+    event.preventDefault();
+    event.stopPropagation();
+    this.props.onSelect(this.props.option, event);
+  }
+
+  handleMouseEnter = (event) => {
+    this.props.onFocus(this.props.option, event);
+  }
+
+  handleMouseMove = (event) => {
```

Comment:
Another small stylistic thing: the parens around `event` can optionally be omitted.

Reply:
The airbnb style uses `()` around a single arg when it's a curly brace function and omits it when it's a single expression.